tests/api_fastapi/execution_api/routes/test_task_instances.py

```diff
@@ -340,6 +340,43 @@ def test_ti_update_state_to_reschedule(self, client, session, create_task_instan
         assert trs[0].map_index == -1
         assert trs[0].duration == 129600
 
+    @pytest.mark.parametrize(
+        ("retries", "expected_state"),
+        [
+            (0, State.FAILED),
+            (None, State.FAILED),
```

Comment:
These should "fail", first one has retries as 0, second one doesn't even have the parameter configured.